help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -61,6 +61,7 @@ Note down the items below before you authenticate to the [!DNL Dynamics 365] CRM
 | `Client ID` | The [!DNL Dynamics 365] Client ID for your [!DNL Azure Active Directory] application. Refer to the [[!DNL Dynamics 365] documentation](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#get-tenant-and-app-id-values-for-signing-in) for guidance. | `ababbaba-abab-baba-acac-acacacacacac` |
 | `Client Secret` | The [!DNL Dynamics 365] Client Secret for your [!DNL Azure Active Directory] application. You would be using option #2 within the [[!DNL Dynamics 365] documentation](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#authentication-two-options). | `abcde~abcdefghijklmnopqrstuvwxyz12345678` for guidance.|
 | `Tenant ID` | The [!DNL Dynamics 365] Tenant ID for your [!DNL Azure Active Directory] application. Refer to the [[!DNL Dynamics 365] documentation](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#get-tenant-and-app-id-values-for-signing-in) for guidance.  | `1234567-aaaa-12ab-ba21-1234567890` |
+| `Region` | Region associated with environment URL.<br> Refer to the [[!DNL Dynamics 365] documentation](https://learn.microsoft.com/en-us/power-platform/admin/new-datacenter-regions) for guidance.| If your domain is as below, you need to provide the highlighted value.<br> *org57771b33.`crm`.dynamics.com*<br> example: For companies provisioned in North America(NAM), select `crm` and for Canada(CAN), select `crm3`|
```

Comment:
Please change: 

As an example: If your company is provisioned in the North America (NAM) region, select `crm`. If your company is provisioned in the Canada (CAN) region, select `crm3`.